IPython/core/display.py

```diff
@@ -512,31 +512,54 @@ def _repr_svg_(self):
 
 
 class JSON(DisplayObject):
-    """JSON expects a JSON-able dict or list
-    
-    not an already-serialized JSON string.
-    
-    Scalar types (None, number, string) are not allowed, only dict or list containers.
-    """
+
     # wrap data in a property, which warns about passing already-serialized JSON
     _data = None
+    def __init__(self, data=None, url=None, filename=None, expanded=True, metadata=None):
```

Comment:
I'm tempted to say that we should default `expanded` to `False` so the amount of data isn't overwhelming. I could go either way (especially since I was the one asking about it).